speaker/bgp_controller.go

```diff
@@ -148,17 +148,17 @@ func hasHealthyEndpoint(eps []discovery.EndpointSlice, filterNode func(*string)
 
 func (c *bgpController) ShouldAnnounce(l log.Logger, name string, _ []net.IP, pool *config.Pool, svc *v1.Service, epSlices []discovery.EndpointSlice, nodes map[string]*v1.Node) string {
 	if !poolMatchesNodeBGP(pool, c.myNode) {
-		level.Debug(l).Log("event", "skipping should announce bgp", "service", name, "reason", "pool not matching my node")
+		level.Warn(l).Log("event", "skipping should announce bgp", "service", name, "reason", "pool not matching my node")
```

Comment:
My biggest concern on this is, this is gonna be spammy if your configuration is legit AND this service shouln't be announced. The networkunabailable one makes sense, because it's an exceptional condition. Also, configuration issues might become easier to spot once https://github.com/metallb/metallb/blob/main/design/crd-status.md lands

Reply:
My cluster doesn't issue any of these "pool not matching my node" messages when in debug mode so I assumed (probably incorrectly) that it would only show if there was a misconfiguration.

I'd be fine with leaving this one as a Debug message then. If the node label message had been a Warn, it would have saved me hours of debugging my cluster last night :)